cpp/src/io/parquet/predicate_pushdown.cpp

```diff
@@ -245,161 +246,21 @@ struct stats_caster {
   }
 };
 
-/**
- * @brief Converts AST expression to StatsAST for comparing with column statistics
- * This is used in row group filtering based on predicate.
- * statistics min value of a column is referenced by column_index*2
- * statistics max value of a column is referenced by column_index*2+1
- *
- */
-class stats_expression_converter : public ast::detail::expression_transformer {
```

Comment:
Declared in `reader_impl_helpers.hpp` now and definitions moved this to the end of this file.